projects/ngx-matomo-client/router/matomo-router.service.ts

```diff
@@ -101,8 +104,8 @@ export class MatomoRouter {
   }
 
   readonly initialize = runOnce(() => {
-    if (this.config.disabled) {
-      // Do not set-up router if globally disabled
+    if (this.config.disabled || isPlatformServer(this.platformId)) {
```

Comment:
Maybe use `!isPlatformBrowser` instead of `isPlatformServer`.

```suggestion
    if (this.config.disabled || !isPlatformBrowser(this.platformId)) {
```

Starting with ng18, there is only browser and server platforms, so this does not change anything.
__But__ in prior versions there were worker-app and worker-ui platforms. This will allow to directly backport this bugfix to previous versions.